crates/core/src/sql/value/value.rs

```diff
@@ -506,17 +506,14 @@ impl Value {
 			Value::Edges(_) => None,
 			Value::Future(_) => None,
 			Value::Constant(_) => None,
-			Value::Function(_) => Some(Kind::Function(None, None)),
+			Value::Function(_) => None,
 			Value::Subquery(_) => None,
 			Value::Query(_) => None,
 			Value::Model(_) => None,
-			Value::Closure(_) => None,
+			Value::Closure(_) => Some(Kind::Function(None, None)),
```

Comment:
Should we also infer the argument list and the return type from the Closure value?

Reply:
Yeah, definitely. Good call.